conduct.md

```diff
@@ -4,35 +4,38 @@
 
 ### Overall Conduct
 
-TODO: Write an introductory paragraph.
+It is expected that the main attitude in the class is to be encouraging and positive, towards yourself and each other. Since this class is mainly based on teams, it is very important that everyone respects each others contribution and provides constructive feedback when necessary. All students are expected to show respect to everyone both inside, and outside the classroom. Students are expected to come to class on time and prepared for collaboration.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Come to every class.
```

Comment:
I'd argue that incomplete phrases like those written here don't warrant a period, but I posted a message on Discord to the whole class to gauge class thoughts on the matter.

Reply:
fixed all of the comments.